### DISCUSSION TITLE
Changing the u() designs to have their actual ride names

### DISCUSSION THREAD

**Discussion by HtotheTML** [OpenRCT2/OpenRCT2#21580] [Feature requests]
Since the names for (almost) all of the u() rides have been datamined and found, I'm wondering if their actual names should replace the placeholder names they currently have.

I have here a spreadsheet with all of the u() designs, along with their backup filenames & their suggested name changes. Some don't have any sort of proper or usable name, so I made a few tweaks here and there to pull everything together cohesively.

https://docs.google.com/spreadsheets/d/1n4wWSwv1LwJIZq5EziJwAsFiHYrSwf69hKJ3vbM_JsQ/edit?usp=sharing

**Comment by jcurbaniak** [comment]
Yes I agree.   I had to rename many rides to get rid of all the U(nnnn) names.

**Comment by HtotheTML** [comment]
Some ride designs are also duplicates of existing ride designs, be they other pre-installed ride designs or ones found in scenarios. I wonder if something could be done about them.

**Reply by Gymnasiast** [reply]
If they’re duplicates of the ones found in scenarios, I think we should leave them in, but give them the name they had in the scenario.

If they’re duplicates of other designs, we can just refuse to load them. Can you provide me with a list?

**Reply by jcurbaniak** [reply]
Thanks, but I would have no idea how to determine duplicates.

**Reply by Gymnasiast** [reply]
This was directed at @HtotheTML , though.

**Reply by HtotheTML** [reply]
I have labeled the duplicates in the spreadsheet accordingly, but here are all of them listed together:
- u(0489) and u(6127) are both identical duplicates of "Rat Flyer" from Wacky Warren
- u(0643) is Quiver from Mel's World
- u(2315) is a copy of the default "Ghost Train" prebuilt design
- u(2448) is a near-perfect copy of "Corkscrew" from Alton Towers, albeit with the station lengthened and the trains recolored
- u(2487) is a duplicate of the "Flying Coaster 1" prebuilt design
- u(4122) is "Claustrophobia" from Diamond Heights.

**Comment by EliasRipley** [comment]
Dunno if you still around to see this but.... didn't know about this thread and mined the same data.

I have 20 cases where we have an exact-SHA256-confirmed name but name suggested on your Google Sheet is something different.

**Reply by HtotheTML** [reply]
Could you clarify which ones you’re referring to?

**Reply by EliasRipley** [reply]
No worries. You were pretty much dead on in reality.
After clearing away extra bloat in names it all made sense. That brought it down to only really 4 differences of something like that...
Made a PR where they are listed with branch editable.
